be/src/storage/tablet_schema.h

```diff
@@ -195,6 +198,7 @@ class TabletColumn {
     ColumnIndexLength _index_length = 0;
     ColumnPrecision _precision = 0;
     ColumnScale _scale = 0;
+    int32_t _referenced_column_id = -1;
```

Comment:
another way does not need add this _referenced_column_id member in TabletColumn, but put this information into RowsetWriterContext.

Reply:
> another way does not need add this _referenced_column_id member in TabletColumn, but put this information into RowsetWriterContext.

change to latter way in 315dd7e43f45959bb03a8317aaf16e93aeff92bb.